src/components/Users/Users.js

```diff
@@ -1,69 +1,71 @@
 import React from 'react';
 
-// import * as axios from 'axios';
+import * as axios from 'axios';
```

Comment:
I think it's possible to just use "import axios from axios".

Reply:
/fxd